docs/user-guides/migrations/includes/src/main/resources/topics/ServerMigration-ServerConfiguration-Subsystem-Update-transactions.adoc

```diff
@@ -0,0 +1,32 @@
+= Update the transaction Subsystem
+
+The JBoss Server Migration Tool makes the following updates to the *transactions* subsystem configuration to better match the {server-target-productName} default configurations.
+
+
+The following attributes were removed from the *transactions* subsystem. The JBoss Server Migration Tool replaces them with the equivalent {server-target-productName} attributes.
+
+[cols="50%,50%", options="header"]
+|===
+|Obsolete Attribute
+|Replacement Attribute
+|path
+|object-store-path
```

Comment:
Such attributes are from the management model, yet the changes must be done at XML level, an example:

`
       <subsystem xmlns="urn:jboss:domain:transactions:1.5">
            <core-environment path="eduardo" relative-to="jboss.server.data.dir">
                ...
            </core-environment>
            ...
        </subsystem>
`

becomes 

`<subsystem xmlns="urn:jboss:domain:transactions:4.0">
            <core-environment>
                ...
            </core-environment>
             ...
            <object-store path="eduardo" relative-to="jboss.server.data.dir"/>
        </subsystem>`